tic_tac_toe/game.py

```diff
@@ -9,8 +11,12 @@ def _position_is_empty_in_board(position, board):
 
     Returns True if given position is empty, False otherwise.
     """
-    pass
-
+    result = None
+    if board[position[0]][position[1]] == '-':
+        result = True
+    else:
+        result = False
+    return result
 
```

Comment:
I think you might be able to remove "result" altogether. So,
15    if board....: 
16        return True
17    else:
18        return False